nix/module.nix

```
@@ -19,7 +19,7 @@ in {
 
     package = mkOption {
       type = types.package;
-      default = pkgs.hyprland or self.packages.${pkgs.system}.default;
+      default = self.packages.${pkgs.system}.default;
```

```suggestion
      default = self.packages.${pkgs.system}.default or pkgs.hyprland;
```

This way we don't break compatibility all the way.